### DISCUSSION TITLE
Help Wanted: New Collaborators

### DISCUSSION THREAD

**Discussion by rewhile** [catppuccin/nvim#261] [Announcements]
# What?

With this discussion I am announcing my retirement from catppuccin.

# Now What?

If you are interested in becoming a collaborator please leave a comment below. We will continue monitoring this thread and make the decision!

Thanks,
Null Chilly

**Comment by nekowinston** [comment]
Many thanks for your work on the port and the graceful exit, @nullchilly! ❤️ 

For people interested in helping out, just let us know here, or feel free to reach out in the [Catppuccin Discord](https://discord.catppuccin.com), for real-time collaboration. 😎

**Reply by vinibispo** [reply]
I want to collaborate

**Reply by nekowinston** [reply]
Hi @vinibispo, would you be able to join our Discord that I linked above?

**Reply by nekowinston** [reply]
I gave you write access; just to clarify, Discord isn't a hard requirement, it's fine if the discussion regarding this port is just on GitHub alone.

**Reply by vinibispo** [reply]
Thanks

**Reply by rewhile** [reply]
@vinibispo If you accepted our invite can you write a unit test? https://github.com/catppuccin/nvim/issues/263

**Reply by vinibispo** [reply]
Yes

**Reply by adityastomar67** [reply]
I would like to collaborate

**Comment by pocco81** [comment]
Indeed, @nullchilly did a lot of great work around the plugin.

New collaborators are, as always, welcome!

**Comment by gotgenes** [comment]
Thanks for your time and energy. I am a minor contributor (#33 #96 ) and use catppuccin as my daily colorscheme (in both Neovim and kitty). I'm always excited to see updates.

Could you say more about what this project needs from a collaborator? A list of some tasks you regularly performed could help give an idea.

I'm sure PR review is one of those responsibilities. Is there anything tricky there?

How does upstream catppuccin/catppuccin impact this project?

Thanks, again.

**Reply by nekowinston** [reply]
Hi @gotgenes, I just gave you write access. 👍  Thanks for your interest in helping maintain the theme!

**Reply by gotgenes** [reply]
Thanks! I'll be joining the Discord server today.

**Comment by pocco81** [comment]
Heya @gotgenes! thanks for your interest in helping around :)

> Could you say more about what this project needs from a collaborator?

+ Review PRs. Particularly enforce the CONTRIBUTING.md rules.
+ If there are new popular plugins coming out then it'd be cool if you add support for them
+ Tackle issues to the best of your abilities, otherwise ask us for help :)
+ Draft a new release in [catppuccin/nvim/releases](https://github.com/catppuccin/nvim/releases)

> I'm sure PR review is one of those responsibilities. Is there anything tricky there?

Tricky? no. But you would always have to be in the lookout for slow code and linear commits.

> How does upstream catppuccin/catppuccin impact this project?

Everything that happens on the main repo should be reflected org-wide. This repo in particular is always the first to be updated and usually does not get affected by automated scripts that would normally update every other repo.

**Comment by adityastomar67** [comment]
I would like to collaborate

**Reply by nekowinston** [reply]
Hi @adityastomar67, I've just given you write access. Thanks for wanting to help out.

**Comment by bashbunni** [comment]
I volunteer as tribute! 😤 
Got too distracted in the discord to reply to the discussion haha

**Reply by nekowinston** [reply]
I've just given you write access to this repository! Many thanks for helping out, good to have you.

**Comment by fioncat** [comment]
I can help! I am using catppuccin in neovim, iTerm2, fzf, tmux, and I like this colortheme very much.
It would be a pleasure for me to become a collaborator.

**Reply by rewhile** [reply]
Invited you, thanks for helping out! ![ ](https://cdn.discordapp.com/emojis/1010154128946778173.webp?size=20&quality=lossless)

**Comment by Fadyio** [comment]
I would like to collaborate

**Comment by mrtnvgr** [comment]
I would love to collaborate!

**Comment by vollowx** [comment]
I'm always free at holidays to collaborate!

**Reply by rewhile** [reply]
Invite sent!

![](https://cdn.discordapp.com/emojis/1012714632227586150.gif?size=240&quality=lossless)
